config/shared_config.go

```diff
@@ -272,6 +280,15 @@ type SharedConfig struct {
 
 	// aws sdk app ID that can be added to user agent header string
 	AppID string
+
+	IgnoreConfiguredEndpoints *bool
```

Comment:
[nonblocking] `bool`?

Reply:
Also doc.

Reply:
i initially had it as a pointer because without it theres not a good way to distinguish between the types zero value and a reasonable value set by the user. (although since false is the default i suppose it may not matter as much). 

additionally, every boolean field type in this file is a `*bool` rather than a `bool`.

im going to keep it for now, but let me know if you disagree

Reply:
That's fine.